api/tests/integration/infrastructure/repositories/granted-accreditation-repository_test.js

```diff
@@ -0,0 +1,74 @@
+const { expect, databaseBuilder } = require('../../../test-helper');
+const grantedAccreditationRepository = require('../../../../lib/infrastructure/repositories/granted-accreditation-repository');
+const { knex } = require('../../../../lib/infrastructure/bookshelf');
+
+describe('Integration | Infrastructure | Repository | granted-accreditation-repository', function() {
+
+  context('#save', function() {
+
+    afterEach(function() {
+      return knex('granted-accreditations').delete();
+    });
+
+    it('should create the granted accreditation', async function() {
+      // given
+      const certificationCenterId = databaseBuilder.factory.buildCertificationCenter().id;
+      const accreditationId = databaseBuilder.factory.buildAccreditation().id;
+      await databaseBuilder.commit();
+
+      // when
+      await grantedAccreditationRepository.save({
+        certificationCenterId,
+        accreditationId,
+      });
+
+      // then
+      const grantedAccreditation = await knex
+        .select('*')
+        .from('granted-accreditations')
+        .where({ certificationCenterId, accreditationId })
+        .first();
+      expect(grantedAccreditation).to.not.be.null;
+    });
+  });
+
+  context('#deleteByCertificationCenterId', function() {
+    it('should delete all granted accreditations for give certification center id', async function() {
```

Comment:
'given'

Reply:
Pour eclaircir ce commentaire: 
```suggestion
    it('should delete all granted accreditations for given certification center id', async function() {
```
😀